src/main/java/trec_eval.java

```diff
@@ -0,0 +1,280 @@
+/*
+* Anserini: A Lucene toolkit for reproducible information retrieval research
+*
+* Licensed under the Apache License, Version 2.0 (the "License");
+* you may not use this file except in compliance with the License.
+* You may obtain a copy of the License at
+*
+* http://www.apache.org/licenses/LICENSE-2.0
+*
+* Unless required by applicable law or agreed to in writing, software
+* distributed under the License is distributed on an "AS IS" BASIS,
+* WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+* See the License for the specific language governing permissions and
+* limitations under the License.
+*/
+
+
+
+import java.io.BufferedOutputStream;
+import java.io.File;
+import java.io.FileOutputStream;
+import java.io.IOException;
+import java.io.InputStream;
+import java.io.InputStreamReader;
+import java.io.OutputStream;
+import java.io.PrintStream;
+import java.lang.ProcessBuilder.Redirect;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+import java.util.Scanner;
+
+import org.apache.commons.io.IOUtils;
+import org.apache.commons.io.LineIterator;
+
+/**
+ * A wrapper for trec_eval native binaries that is usable on a number
+ * of platforms.
+ * <br><b>Supported Platforms</b><br>
+ * <ul>
+ * <li>Windows (Intel 32bit)</li>
+ * <li>Linux (Intel 32bit)</li>
+ * <li>Mac OS X (Intel 64bit)</li>
+ * </ul>
+ * @author Craig Macdonald
+ */
+public class trec_eval 
```

Comment:
Can you change to two-space indent, just like all our other code?
Opening braces go on previous line.